modules/optimization/test/tests/optimizationreporter/general_opt/point_loads_gen_opt/tests

```diff
@@ -0,0 +1,55 @@
+[Tests]
+  issues = '#26476 #21885 #24101 #25689'
+  design = 'GeneralOptimization.md OptimizationData.md ReporterPointSource.md'
+  [gen_opt_point_loads]
+    requirement = "The system shall be able to invert for point loads using the general optimization reporter with"
+    [no_measurement_transfer]
+      type = CSVDiff
+      input = main_auto_adjoint.i
+      csvdiff = 'main_out_OptimizationReporter_0001.csv'
+      max_threads = 1 # Optimize executioner does not support multiple threads
+      rel_err = 0.1
+      abs_zero = 10
```

Comment:
what s up with those?
this is acception ANY result pretty much

Reply:
This PR is demonstrating a new method to setup the optimization problem. I recreated a set of tests the optimization module and we had the tolerances set to those values. It's set to those values to only check the temperature but not the gradients. Comment below about setting them.
https://github.com/idaholab/moose/pull/26486#discussion_r1443191312

Reply:
This is not how we do partial comparisons if you want to ignore the gradients.
Do this instead:
https://mooseframework.inl.gov/python/testers/Exodiff.html#a4141dea-5304-4be6-bd63-fd630fe95ff9